### PR TITLE
add qa1_nodes_reboot script

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

**Comment by kbaikov** [comment]
+1 

Tested on qa1 hardware. Looks fine to me. qa1 nodes were restarted and discovered.

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1254,6 +1254,9 @@ function onadmin_allocate()
         done
         wait
     fi
+    if [[ $cloud = qa1 ]] ; then
+        curl http://clouddata.cloud.suse.de/git/automation/scripts/qa1_nodes_reboot | bash
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: :-1: 
rather like this (thats exactly what we have these variables for, please use them):
```
${mkcloud_dir}/qa1_nodes_reboot
 or
bash -x ${mkcloud_dir}/qa1_nodes_reboot
```

**Comment by toabctl** [comment]
Publishing our internal credentials on github isn't nice. What about using some ENV vars or reading username/password from a file? Beside that, looks good to me.

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: There is no mkcloud_dir var in qa_crowbarsetup ... and possibly no git checkout either.

**Comment by jdsn** [comment]
I regard this password as equally secure as 'linux' or 'crowbar' and we use them in many places :)

What about storing the password on clouddata and fetching it from there?

**Comment by bmwiedemann** [comment]
Changed it so that the PW is provided at runtime

**Comment by jdsn** [comment]
:+1:

**Pull request merged**